examples/reflection/reflection_types.rs

```diff
@@ -80,9 +80,13 @@ fn setup() {
         // arity 12 or less.
         ReflectRef::Tuple(_) => {}
         // `List` is a special trait that can be manually implemented (instead of deriving Reflect).
-        // This exposes "list" operations on your type, such as indexing and insertion. List
-        // is automatically implemented for relevant core types like Vec<T>
+        // This exposes "list" operations on your type, such as insertion. `List` is automatically
+        // implemented for relevant core types like Vec<T> and [T; 15].
```

Comment:
Is this accurate? I thought arrays specifically did not implement List, since they're not resizable?

Reply:
Ah good catch, I didn't realize this was in regards to `List` and not `Array`. I'll fix that.